### PR TITLE
fix: Additional PE-D UG improvements

### PR DESCRIPTION
**Pull request by jayllo-c**
Currently, as the testers from the PE-D have pointed out, the UG has
room for improvement in regards to the clarity of some of the commands (`importExamScores` and `find`) as well as the omission of usage 
information for some prefixes (`lt|` and `mt|`).

This has to be fixed because future users would likely be unclear in
the purpose behind some of the features of the application.

Let's revamp the UG to include the missing use cases and improve the
clarity of the user guide.

In doing so, future users will have a better idea of the expected
behaviour of the app and the rationale of why the behaviour of the app
was designed as such.

### CODE REVIEW

**Review by delishad21 — COMMENTED** [review]
Small changes that helps our users understand our application better. Well done!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -102,6 +102,9 @@ Click below to navigate the user guide:
 
 ## Contact Management Features
 
+These features are designed to help you manage your contacts effectively and centralizes all contact information in one place.
```
Comment (danielleloh) [inline docs/UserGuide.md:105]: nice addition :)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -413,13 +416,22 @@ Filters your contacts based on specific criteria you set.
 
 **Information:** <br>
 * Use this command to search for persons using a specific aspect of their details, as specified by the prefix.
-* The search will return any result that contains the keyword you have specified.
-    > e.g. `find e|john` will find any person that contains `john` in their email.
+* The search will return any result that contains the keyword you have specified or matches the condition provided by the user.
+    > e.g. `find e|john` will find any person that contains `john` in their email. <br>
+    > e.g. `find lt|50` will find any person who scored lower than 50 marks for the selected exam. <br/>
```
Comment (danielleloh) [inline docs/UserGuide.md:421]: good addition to show users how to use the lt and mt parameters

**Review by danielleloh — COMMENTED** [review]
nice additions to the UG

**Pull request merged**